src/components/Confirm/index.js

```diff
@@ -84,15 +84,16 @@ const Previews = ({capture, retakeAction, confirmAction} ) =>
 
 
 export default class Confirm extends Component {
-  completeCapture(method, side, capture, confirmCapture, nextStep) {
+  completeCapture(id, method, side, capture, confirmCapture, nextStep) {
+    capture.id = id
     confirmCapture({method, id: capture.id})
     confirmEvent(method, side)
     nextStep()
   }
 
   startApiUpload(method, side, capture, token, confirmCapture, nextStep, advancedValidation, onApiUpload, onApiError) {
     onApiUpload()
-    postToOnfido(capture, method, token, advancedValidation, () => this.completeCapture(method, side, capture, confirmCapture, nextStep), onApiError)
+    postToOnfido(capture, method, token, advancedValidation, ({id}) => this.completeCapture(id, method, side, capture, confirmCapture, nextStep), onApiError)
```

Comment:
Can you shorten this line to c. 80 characters?

I also don't like implicit unpacking. `onSuccess` gets passed a JSON response object so I would call the argument of this function `jsonResponse`. Then you can get the ID with `jsonResponse.id`, which is much more readable